packages/editor/src/components/rich-text/index.js

```diff
@@ -1056,17 +966,12 @@ const RichTextContainer = compose( [
 	withSafeTimeout,
 ] )( RichText );
 
-RichTextContainer.Content = ( { value, format, tagName: Tag, ...props } ) => {
-	let content;
-	switch ( format ) {
-		case 'string':
-			content = <RawHTML>{ value }</RawHTML>;
-			break;
-
-		case 'children':
-			content = <RawHTML>{ children.toHTML( value ) }</RawHTML>;
-			break;
-	}
+RichTextContainer.Content = ( { value, tagName: Tag, multiline, ...props } ) => {
+	const content = (
+		<RawHTML>
+			{ toString( value, multiline ) }
```

Comment:
Kind of aligns with the existing format the is for `RichText` which is just called `string`, but I don't really care that much here.

Reply:
Ah, okay. I mean I think consistency is probably better than selectively naming it… and changing the `RichText` format would be breaking, right? If so I think it's fine to leave, but if it's easy to change _everywhere_ it might make things a bit clearer? 🤷‍♂️

Reply:
Sure, I'm fine with renaming it `toHtmlString`. Should `HTML` not be lowercased here? I remember your last comment that the `HTML` variable should better be `html` lowercased? So isn't this similar?

Reply:
Soooooo that one is really annoying because I think HTML, as an abbreviation, should actually be uppercased **unless** it's a variable that should *start* with a lowercase "h", which when it's a non-constant variable. So I think it's:

```
// Good
const html = '<p>Hi</p>';

// Good
export default class HTML extends Component {}

// Good
function toHTML() {}
```

![as-sigh](https://user-images.githubusercontent.com/90871/45094078-2d292900-b112-11e8-8a52-2483734ef60f.gif)

----

Does that make sense? Sorry 😩